src/js/manager.js

```diff
@@ -42,13 +55,16 @@ class SmolLinksManager {
 
 			let form = document.querySelector('.smol-links-create');
 			form.addEventListener('submit', this.createShlink.bind(this));
+
+			let pagination = document.querySelector('.smol-links-pagination');
+			pagination.addEventListener('change', this.updatePage.bind(this))
 		} catch (err) {
```

Comment:
this catch doesn't seem to be doing anything now, can it disappear?

Reply:
ah thanks, fixed it!